src/Components/Blazor/Build/src/Tasks/BlazorILLink.cs

```diff
@@ -114,6 +116,11 @@ protected override string GenerateResponseFileCommands()
                 }
             }
 
+            if (!string.IsNullOrEmpty(FrameworkAssemblyAction))
+            {
+                args.Append("-c ").AppendLine(FrameworkAssemblyAction);
+            }
```

Comment:
Does this change come from the mono/linker repo? I'm guessing we don't want to fork the code for this task if we can avoid it, since later on we want to switch over to use the SDK version of this task.

Reply:
They have a feature to do this per assembly, but I could see us porting this back to the .NET SDK.